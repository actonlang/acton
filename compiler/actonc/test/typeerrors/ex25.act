actor Bob():
    def foo(cb):
        pass

actor Rob():
    def bar():
        b = Bob()
        b.foo()
