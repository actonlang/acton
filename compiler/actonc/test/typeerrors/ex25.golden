Building file test/typeerrors/ex25.act using temporary scratch directory
  Compiling ex25.act for development
[error]: Incompatible types
     +--> ex25.act@8:9-8:16
     |
   8 |         b.foo()
     :         ^------
     :         `- keyword component(s) 'cb' is missing in tuple
-----+
